### ISSUE TITLE
Ticked closed without confirmation/disconfirmation

### ISSUE THREAD

**Issue by Mangonels** [DecentSoftware-eu/DecentHolograms#61]
### Just making sure

- [X] I have read the wiki and made sure, this feature is not yet in the plugin.
- [X] I believe, that this feature is possible to implement.

### Problem

https://github.com/DecentSoftware-eu/DecentHolograms/issues/53

### Feature

My request was closed, without further comments, not sure if it was completed (it doesn't seem like it) or just closed because there is no implementation intentions.

I do believe implementing this is technically posible.

One of the arguments against this idea was the imposibility of click detection with certain entities. Not every context requires click detection, so I believe this is not an issue.

**Comment by Andre601** [comment]
Better click detection was added in v3

**Comment by Mangonels** [comment]
> Better click detection was added in v3

My ticket doesn't really refer to click detection as a request. The actual request is for base entity modifications (please refer to the attatched issue #53).

I'm not entirely sure what you are trying to convey with that statement.

**Comment by d0by1** [comment]
Hello, your issue was closed because the changes you suggested will, to some extent, be added in the upcoming v3 update. Current version of the plugin, v2, will no longer be recieving any new features so I closed the issue and put your suggestion on my to-do list. 

Yes, the new click detection doesn't use the base entities and is done differently. 

Hiding entities in GM3 is not really possible without changing the entity type to something like the particles you suggested. I will try to achieve this and possibly make it a feature in v3 as well. 

Hope you understand now. Thanks for your suggestions!
